src/search.ts

```diff
@@ -12,7 +13,10 @@ import { HitDecorator, Options, DecoratedProduct, DecoratedResult } from "./type
 export async function search<HD extends HitDecorator[]>(query: SearchQuery, options: Options<HD> = {}) {
   const { hitDecorators, ...rest } = options
   const api = await new Promise(nostojs)
-  return applyDecorators(await api.search(query, rest), hitDecorators)
+  const endMeasurement = measurePerformance("nosto.search")
+  const searchResult = await api.search(query, rest)
+  endMeasurement()
```

Comment:
should this include the decoration?
Also what is the relation of nosto.search to performance measurements in the client script regarding the search call?
this could also be applied there